cmake/O2Dependencies.cmake

```diff
@@ -136,6 +136,7 @@ o2_define_bucket(
     DEPENDENCIES
     Base
     Headers
+    DataFlow
```

Comment:
not sure about this, comes probably from the fact that SubframeMetadata.h is in the wrong place